### ISSUE TITLE
[CI] 7.x FullClusterRestartSettingsUpgradeIT.testRemoteClusterSettingsUpgraded failing

### ISSUE THREAD

**Issue by hendrikmuhs** [elastic/elasticsearch#49587]
Many failures on different version combinations, e.g.

https://gradle-enterprise.elastic.co/s/dwtot5uhqr2we

reproduces on 7.x with

```
./gradlew ':qa:full-cluster-restart:v6.1.0#upgradedClusterTest' --tests "org.elasticsearch.upgrades.FullClusterRestartSettingsUpgradeIT.testRemoteClusterSettingsUpgraded" -Dtests.seed=3F32EC744B60978D -Dtests.security.manager=true -Dtests.locale=bg -Dtests.timezone=Africa/Abidjan
```

If I am not wrong its a regression of https://github.com/elastic/elasticsearch/commit/416178c7c8644ea9ebee45a33fc680d6b642ab99

I am muting this test.

**Comment by hendrikmuhs** [comment]
muted: https://github.com/elastic/elasticsearch/commit/41daf284f5b2e6faeb6b6d58a5095a21ca3aeba2

**Comment by ywelsch** [comment]
Closed by #49609
